packages/components/src/ListingDetailPhaseCard/ChallengeResults.tsx

```diff
@@ -83,7 +82,7 @@ export class ChallengeResults extends React.Component<ChallengeResultsProps> {
             <VotesPerTokenVote vote="remain">
               <span>✔</span> Remain
             </VotesPerTokenVote>
-            <VotesPerTokenCount>{getNumberStringWithCommaDelimeters(this.props.votesAgainst)} CVL</VotesPerTokenCount>
```

Comment:
should this just use `getFormattedTokenBalance`?

Reply:
Yeah, I removed this method call... and `getFormattedTokenBalance` gets called in the views that implement this view. I'm going to optimize and move those helper calls to this view, but I think I'll do that in a near-future update.